ovos_PHAL_plugin_mk1/__init__.py

```diff
@@ -589,3 +594,62 @@ def on_weather_display(self, message=None):
                 icon = "x=2," + icon
                 msg = "weather.display=" + str(temp) + "," + str(icon)
                 self.writer.write(msg)
+
+    # date/time
+    def on_display_date(self, message=None):
+        self._deactivate_mouth_events()
+        self.on_text(message)
+        sleep(10)
+        self.on_display_reset()
+        self._activate_mouth_events()
+
+    def on_display_time(self, message=None):
+        code_dict = {
+            ':': 'CIICAA',
+            '0': 'EIMHEEMHAA',
+            '1': 'EIIEMHAEAA',
+            '2': 'EIEHEFMFAA',
+            '3': 'EIEFEFMHAA',
+            '4': 'EIMBABMHAA',
+            '5': 'EIMFEFEHAA',
+            '6': 'EIMHEFEHAA',
+            '7': 'EIEAEAMHAA',
+            '8': 'EIMHEFMHAA',
+            '9': 'EIMBEBMHAA',
+        }
+
+        if message and message.data:
```

Comment:
this should be impossible to reach without a message object, if its triggered via bus the message is always there